Products/DataCollector/SshClient.py

```diff
@@ -88,6 +89,8 @@ def hasattr_path(object_root, path):
         else:
             log.debug("Couldn't get the remote device's hostname")
 
+    log.debug('Device name is: "%s"', device)
+
```

Comment:
This looks like some left over debugging code?

I think it should be removed.  The debug logging for sending events will log the event which has the device name in it.